### DISCUSSION TITLE
Viết báo cáo gửi thầy!!!

### DISCUSSION THREAD

**Discussion by baonhu1304** [baonhu1304/Lop-10A2-nhom-1#10] [Announcements]
Mọi ngừi uiii,
Tình hình là chiều nay tụi mình phải nộp báo cáo cho Thầy Hiền đẹp troai á. Nên mọi người ghi chú nguồn/ link web ở column tên mình để các bạn khác tham khảo hoặc viết báo cáo cho dễ nha.
Cảm ơn mọi người nhìu nghennn

**Comment by Lenga10A2** [comment]
Ủa tui thấy trên teams thầy còn kêu làm cái file word j đó nữa là sao á bà


Vào 11:47, T.5, 14 Th10, 2021 Phạm Bảo Như ***@***.***> đã
viết:

**Reply by baonhu1304** [reply]
Uh cái file đó đó bà. Nhưng mà file đó là mỗi người đều điền hay đại diện taa??

**Comment by Lenga10A2** [comment]
Ừ đó tui cx thắc mắc á

Vào 14:41, T.5, 14 Th10, 2021 Phạm Bảo Như ***@***.***> đã
viết:

**Comment by thaopham2501** [comment]
Như ơi, tui sửa xong ròi á ::v có gì viết báo cáo đi :')
